src/Markdig/Syntax/Inlines/LinkInline.cs

```diff
@@ -3,6 +3,7 @@
 // See the license.txt file in the project root for more information.
 
 using Markdig.Helpers;
+using Markdig.Parsers;
```

Comment:
Why this is added? - same above -

Reply:
Oh, maybe for XML documentation?

Reply:
Also for XML documentation. I double-checked, and it was only needed for a reference to `MarkdownParser.TrackTrivia`. Since that class is never actually exposed to the user as an instance, I changed these to point to `MarkdownPipeline.TrackTrivia` instead.